src/WooCommerce.php

```diff
@@ -249,6 +249,19 @@ public static function cron_remove_back_in_stock() {
     }
   }
 
+  /**
+   * Cron event callback to remove outdated back-in-stock product metadata.
+   */
+  public static function cron_orphans_variants_cleanup() {
+    global $wpdb;
+    $wpdb->query(
```

Comment:
Query taken of out the method use by the wc cli: https://github.com/woocommerce/woocommerce/blob/3a76aa42555f037a0a565e9f0feaa43f89b197d2/plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-system-status-tools-v2-controller.php#L464

We could also instantiate` WC_REST_System_Status_Tools_V2_Controller` can call execute_tool(''delete_orphaned_variations")